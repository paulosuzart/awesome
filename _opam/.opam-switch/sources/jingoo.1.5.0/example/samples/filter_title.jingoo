{{ 'this phrase should be in title case!' | title }}
