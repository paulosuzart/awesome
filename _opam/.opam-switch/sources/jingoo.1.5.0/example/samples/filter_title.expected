This Phrase Should Be In Title Case!
